### PR TITLE
Remove nose

### PR DESCRIPTION
**Pull request by tfarago**

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    slow: marks tests as slow (deselect with '-m "not slow"')
+    skip_ci: do not run the test on CI platforms
```
Comment (tfarago) [inline pytest.ini:4]: We don't need it right now but might want it in the future.

**Comment by tfarago** [comment]
The event loop warnings will be fixed in #494

**Comment by MarcusZuber** [comment]
> @MarcusZuber can you take care of the zero divisions [e.g. here](https://github.com/ufo-kit/concert/blob/master/concert/experiments/addons.py#L323)?

Sure. Do you mean for arbitrary data or the artificial test case?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ jobs:
         pip install scikit-image
     - name: Test with pytest
       run: |
-        coverage run --source=concert -m nose --attr='!skip-ci'
+        coverage run --source=concert -m pytest -m "not skip_ci"
```
[the remark this answers is not in the document]
Comment (tfarago) [reply .github/workflows/tests.yml:37]: Sure, but something tells me it's not going to be blazing fast... We can also skip all `slow` tests, which used to be the case back then if my memory serves me well. On the other hand, you can have a quick coffee in those 3 min.

**Comment by tfarago** [comment]
> > @MarcusZuber can you take care of the zero divisions [e.g. here](https://github.com/ufo-kit/concert/blob/master/concert/experiments/addons.py#L323)?
> 
> Sure. Do you mean for arbitrary data or the artificial test case?

Of course _any_ input must be processed correctly and zero division taken care of by either disregarding it or raising an exception.

**Comment by MarcusZuber** [comment]
Division-by zero warning is now covered in grating interferometry addon.

I had a look at other parts in the code and was surprised, that you put the inf in the flatfield correction to zero.

File: .github/workflows/tests.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (MarcusZuber) [reply .github/workflows/tests.yml:37]: But I already drink way to much coffee...

I would keep them in here. Most of the slow ones are doing something with timing and sleeps, so I'm a bit confident, that they can be a bit faster if executed in parallel. Just give it a try and then we decide.

[the remark this answers is not in the document]
Comment (MarcusZuber) [reply .github/workflows/tests.yml:37]: On my local PC the tests run for 105s. With 8 workers it takes ~30s. I don't know how many threads github provides.

**Comment by MarcusZuber** [comment]
Exchanging LOG.warn by LOG.warning reduces coverage :confused: ?

**Comment by tfarago** [comment]
> Exchanging LOG.warn by LOG.warning reduces coverage confused ?

No, where do you see it?

**Comment by MarcusZuber** [comment]
> > Exchanging LOG.warn by LOG.warning reduces coverage confused ?
> 
> No, where do you see it?

https://app.codecov.io/gh/ufo-kit/concert/compare/496/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr%20comments&utm_term=ufo-kit#diff-Y29uY2VydC9iYXNlLnB5

**Comment by tfarago** [comment]
From my point of view this is done. I don't mind waiting for the tests and would rather have complete test suite ran than skipping the slow ones and then revealing problems at the beam line. However, you can convince me. You can also use the additional `... and not slow` in the workflow and then switch it back when your PR is ready.

**Review by MarcusZuber — APPROVED** [review]

**Comment by tfarago** [comment]
> No, I think there was a misunderstanding. Leave as many tests in as possible. It would have been nice, if the slow ones would be faster, but I want them to be tested in any case.

I think that can be worked on a little bit, but outside of this PR.

**Pull request merged**